agents/processor/src/push.rs

```diff
@@ -41,12 +39,9 @@ impl std::fmt::Debug for Pusher {
 
 impl Pusher {
     /// Instantiate a new pusher with a region
-    pub fn new(name: &str, bucket: &str, region: Region, db: NomadDB) -> Self {
-        let client = S3Client::new_with(
-            HttpClient::new().unwrap(),
-            EnvironmentProvider::with_prefix(AWS_S3_PREFIX),
-            region.clone(),
-        );
+    pub fn new(name: &str, bucket: &str, db: NomadDB) -> Self {
+        let region: Region = Default::default(); // loads from aws env
+        let client = S3Client::new(region.clone());
```

Comment:
Did you run this by @arnaud036? Would need to add env var

Reply:
yep! we're trying to move towards more industry-standard AWS configuration. this is part of that :)